src/js/base/Color.tsx

```diff
@@ -46,79 +43,57 @@ const colors: string[] = [
     "EC4899",
 ];
 
-const StyledColor = styled(Box)`
-    padding: 10px;
-`;
-
 type ColorSquareProps = {
-    className?: string;
+    // The color in hex format
     color: string;
+    // The callback to be called when the color is clicked
     onClick: (color: string) => void;
 };
 
-const UnstyledColorSquare = ({ className, color, onClick }: ColorSquareProps) => {
+/**
+ * A color square that updates the color input when clicked
+ */
+function ColorSquare({ color, onClick }: ColorSquareProps) {
     const handleClick = useCallback(() => onClick(color), [color, onClick]);
-    return <button className={className} type="button" title={color} onClick={handleClick} />;
-};
-
-const ColorSquare = styled(UnstyledColorSquare)<ColorSquareProps>`
-    backface-visibility: hidden;
-    background-color: ${props => props.color};
-    background-image: none;
-    border: none;
-    cursor: pointer;
-    flex: 1 0 auto;
-    width: auto;
-    transform: translateY(0);
-
-    &:first-child {
-        border-bottom-left-radius: ${borderRadius.sm};
-        border-top-left-radius: ${borderRadius.sm};
-    }
-
-    &:last-child {
-        border-bottom-right-radius: ${borderRadius.sm};
-        border-top-right-radius: ${borderRadius.sm};
-    }
 
-    &:focus {
-        box-shadow: rgb(0 0 0 / 25%) 0 0 5px 2px;
-        outline: solid white 2px;
-        z-index: 2;
-    }
-
-    &:hover {
-        transform: translateY(-2px);
-    }
-`;
-
-const ColorGrid = styled.div`
-    align-items: stretch;
-    background-color: transparent;
-    display: flex;
-    height: 30px;
-    margin-top: 10px;
-`;
+    return (
+        <button
+            type="button"
+            title={color}
+            onClick={handleClick}
+            className={
+                "flex-1 h-full transition-transform hover:-translate-y-0.5 focus:outline-none focus:ring-2 focus:ring-white focus:ring-offset-2 focus:ring-offset-gray-300 focus:z-10 first:rounded-l-sm last:rounded-r-sm"
+            }
+            style={{ backgroundColor: `#${color}` }}
```

Comment:
Put colour into classNames and keep it as names rather than hex format. This is also done in `Label` so you can use that as a template

Reply:
Should I still keep a mapping between tailwind colors and the hex codes? When I replace this with just the names as in the `Label` component, the API calls fail because the format of the color code is invalid

Reply:
I pushed this change so you can see the error too

Reply:
The color has to be hex in this case. This is a clearer solution than putting it in a CSS class.
```js
style={{ backgroundColor: `#${color}` }}
```